util/Cargo.toml

```diff
@@ -19,3 +19,4 @@ fdo-http-wrapper = { path = "../http-wrapper", version = "0.5.0", features = ["s
 serde_yaml = "0.9"
 serde_cbor = "0.11"
 serde_json = "1"
+sysinfo = "0.30.5"
```

Comment:
Can this functionality made a dev dependency, or will they be used for things other than tests?

Reply:
Currently this is used in tests only. Not sure if it will be used or not in other parts of the code in the future though.

Reply:
Actually is used in `integration-tests` which is not part of the unit tests, so I guess it cannot be moved to `[dev-dependencies]`

Reply:
I have removed sysinfo dependency as per @7flying comments